Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -422,6 +422,11 @@ public static Furniture PlaceInstance(Furniture proto, Tile tile)
         return obj;
     }
 
+    public bool WorkspotIsInternal()
```

Comment:
`Jobs` is an instance of `BuildableJobs`. `BuildableJobs` holds everything Job related of the furniture. Was called `FurnitureJobs`, but changed it when I added them to the Utilities, since that is the Interface name.